src/stories/containers/finances-overview/useFinancesOverview.ts

```diff
@@ -1,10 +1,22 @@
 import { useThemeContext } from '@ses/core/context/ThemeContext';
+import { DateTime } from 'luxon';
+import { useState } from 'react';
 
 const useFinancesOverview = () => {
+  const actualYear = DateTime.local().minus({ year: 1 }).year;
+  const [year, setYear] = useState<number>(actualYear);
```

Comment:
you can combine these two lines as follow, so the new date is not created on every render:
```js
const [year, setYear] = useState<number>(() => DateTime.local().minus({ year: 1 }).year);
```

Reply:
done